### PR TITLE
Allow uses to shutdown the pprof

### PR DESCRIPTION
**Pull request by astaxie**
pprof looks like a debug tools. So I think the default should turn off.
And if pporf turn on, user can also change the pprof addr.

### CODE REVIEW

**Comment by cch123** [comment]
谢大真是勤奋

**Comment by coocood** [comment]
Thank you for your contribution.

Being able to change the address and having the option to turn it off is great,  but since TiDB is under heavy development, having Debug set to true by default would be more convenient.

I also noticed that the PR is sent from your master branch, we recommand that when opening a PR, create a topic branch and make changes on that branch, it's easier for your forked repo to keep in sync with upstream.

**Comment by astaxie** [comment]
> having Debug set to true by default would be more convenient.

HERE WE GO https://github.com/pingcap/tidb/commit/7adf722a535e25c6abc0e605439380b981054e3d
:smiley: 

> create a topic branch and make changes on that branch, it's easier for your forked repo to keep in sync with upstream.

I think it just make sense from branch name. but it's not related to the upstream.

**Comment by coocood** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

File: tidb.go [hunk 1]
```diff
@@ -74,7 +73,9 @@ var (
 	domap = &domainMap{
 		domains: map[string]*domain.Domain{},
 	}
-	stores = make(map[string]kv.Driver)
+	stores    = make(map[string]kv.Driver)
```
Comment (xiang90) [inline tidb.go]: Since TiDB can be used as a library, having these global variable are inconvenient if not dangerous.

Comment (astaxie) [inline tidb.go]: But we already has the `RegisterLocalStore` and `RegisterStore` for user to register the stores. As there still has some valid in these functions.

Comment (xiang90) [inline tidb.go]: That is why I did not comment on your change. We need to change all of them.

**Pull request merged**